frontend/src/component/admin/adminRoutes.ts

```diff
@@ -3,45 +3,74 @@ import type { INavigationMenuItem } from 'interfaces/route';
 export const adminGroups: Record<string, string> = {
     users: 'User administration',
     access: 'Access control',
+    sso: 'Single sign-on',
+    network: 'Network',
     instance: 'Instance configuration',
-    log: 'Logs',
-    other: 'Other',
 };
 
 export const adminRoutes: INavigationMenuItem[] = [
+    // Admin home
```

Comment:
Moved all the original routes to oldAdminRoutes.ts - which will be deleted when we remove the flag